services/lumberjack/package-lock.json

```diff
@@ -0,0 +1,417 @@
+{
```

Comment:
This should be removed, other services don't have package-lock.json files committed.